### ISSUE TITLE
Privacy Policy

### ISSUE THREAD

**Issue by HarryHeights** [AllanWang/Frost-for-Facebook#1418]
HarryHeights commented 20 minutes ago
Hello, I'm Harry. I engage in creating documents for open source projects. It's part of my contribution in helping open source projects. One of the documents I engage in creating is a privacy policy. The privacy policy will give details about user information collection, use and security. I really don't know if you externally collect user information, but, if you don't, it's good to let end users know via a privacy policy. I'll love to hear your thoughts on this.

**Comment by AllanWang** [comment]
I have a loose policy in layman's term [here](https://github.com/AllanWang/Frost-for-Facebook/blob/dev/docs/Privacy.md). I basically don't collect anything identifiable unless it is explicitly sent.

I used to have a more formal privacy policy, but I have since discarded that. This project is still under the [gpl-v3 license](https://github.com/AllanWang/Frost-for-Facebook/blob/dev/LICENSE#L45), which states no warranty or liability on my part through the usage of this application.

Contributions are also loosely defined. This isn't my main project, and I do not have a road map that is set in stone. I document and categorize my issues, and anyone can attempt to solve them. If the PR looks good and is verifiable, I will merge it. This project does use Zenhub, so users can see my scrum board if they wish (which I somewhat follow)
